### ISSUE TITLE
Didn't work on my OP8P or my OP7P

### ISSUE THREAD

**Issue by MitchRapp01** [Wishmasterflo/Firmware_flasher#13]
OP7P would only boot into bootloader and OP8P got stuck on the logo flashing white to red but never made into into system.  Tried multiple firmware for both phones.  Not sure why it wouldn't work.  Would like it to because this is so convenient.

**Comment by Wishmasterflo** [comment]
Are you on Stock OOS or a custom ROM. Never heard that someone got issues with phone not booting like that.
Sounds more like you are on Srock OOS and did not flash the same OOS Firmware as your OOS Version.

**Comment by MitchRapp01** [comment]
I'm on stock oos. The thing that might be the problem is that the phones had erased themselves and there was no os when I went to flash the firmware. Is that why it didn't work?

**Comment by Wishmasterflo** [comment]
Yes, if you just flash the Firmware and dont have a proper OOS Version or custom ROM which is based on that Firmware installed before you boot your phone to system it will not boot.
So just flashing the Firmware will not make your phone to boot to anything if you dont have a proper OOS or custom ROM installed for that Firmware.
